app/assets/stylesheets/admin/admin.sass

```diff
@@ -0,0 +1,47 @@
+// -----------------------------------------------------------------------------
+// This file contains all Admin styles.
+// -----------------------------------------------------------------------------
+
+@import "normalize-rails"
+@import "bourbon"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```

Comment:
Actually, to be more more accurate:

`//` = internal comment that will never appear in any form of compiled css
`/*` = comment will appear in compiled css but will get stripped out by minifiers
`/*!` = comment will appear in compiled css but will **NOT** get stripped out by minifiers